### PR TITLE
Changes since 022715

### PR DESCRIPTION
**Pull request by moxiegirl**
There have been a lot of changes since Feb 27, this catches us up. It also includes some line wrap updates from @thaJeztah to the birthday material.  Since the last PR had an issue building. I'm resubmitting from a new branch as a precaution.

### CODE REVIEW

**Comment by thaJeztah** [comment]
Looks like this one is complaining as well on commits not having a proper DCO.

Either this is because of the "cherry-pick" or (ahum) some maintainers forgot to sign off (the first ones at least are commits by maintainers). This are the commits that are failing;

https://github.com/moxiegirl/docker/commit/e0b1a8a2e22ba3f05bf51aa79bd27df91af9d1b7
https://github.com/moxiegirl/docker/commit/99605b5ed5378f75ad333d79bb2a18ff4df8f5db
https://github.com/moxiegirl/docker/commit/0ea2f959c19d953a785e522f98afe7503ae5a82a
https://github.com/moxiegirl/docker/commit/7263bfd36f8fb8702fe17de585f2d8342d0cca79
https://github.com/moxiegirl/docker/commit/413d7a2947738d6a1f0d9934330ae1560160f82d
https://github.com/moxiegirl/docker/commit/a3191bf213438bea6f3f1fbfd3a9e953344df094
https://github.com/moxiegirl/docker/commit/0d8938b67a876005cfecd5d9b32c65c6cfe6913e
https://github.com/moxiegirl/docker/commit/752830c7e1dfbe6225f9ffd38387d727b81efc1a
https://github.com/moxiegirl/docker/commit/e1b346d053a0b86216becbeb6aef184cf962b493
https://github.com/moxiegirl/docker/commit/5f621cbdef9ee8098ac84680259393e55ae09540
https://github.com/moxiegirl/docker/commit/eb6428f11513d2bda0689ad32de02aa3fcf74fb8
https://github.com/moxiegirl/docker/commit/ed1bba53b86ffe813373904d92f3afec1bded7e8
https://github.com/moxiegirl/docker/commit/a9478462b1b7fbc6caa8331a22068ae336c04974
https://github.com/moxiegirl/docker/commit/bb1b3d65a1933b6e3177f54e720ade651cca80b5
https://github.com/moxiegirl/docker/commit/576a0c6429f918f7a1e811a6a3203c09730d09a0
https://github.com/moxiegirl/docker/commit/05de1d50d890b76b14744d1eee4262757ff63728
https://github.com/moxiegirl/docker/commit/1a2220a0f241880ca92e6ea77ca177abae5bd193
https://github.com/moxiegirl/docker/commit/2099756edcdef34509652f4fb3f8896ae40f99be
https://github.com/moxiegirl/docker/commit/b7af19da21be6d73a8cba249c861c49365c000e7
https://github.com/moxiegirl/docker/commit/28e8a20badddc2e45d1c5136c35884cbba2bedf2
https://github.com/moxiegirl/docker/commit/ff9512c631f50786566e88e03a43d77a53468a21
https://github.com/moxiegirl/docker/commit/2f37cddd4fc1e064a4555f7875d03d64dde28dee
https://github.com/moxiegirl/docker/commit/3e85eaf4b8a94325b7a71d57c33acd949e0878d0
https://github.com/moxiegirl/docker/commit/5e1d20ca201ed811ec40b358e89833334e5ed0b6
https://github.com/moxiegirl/docker/commit/5b1bce410742c41cdb9ee7a133ffd96a9bc85a7a
https://github.com/moxiegirl/docker/commit/fe185ea47c9e08a6e094b3104a7b322213c32d60
https://github.com/moxiegirl/docker/commit/4e466babf41b3ffee03bc4dad064b92e9567ecc9

**Comment by moxiegirl** [comment]
@thaJeztah Yes, I know. I sent a note to Jess about this.  These commits are not the ones I cherry picked.  I checked that.  I even, as you'll notice, redid a new pull request to hopefully fix the issue...t y p i n g    e v e r y t h i n g slowly to rule out operator error.  I need more information and it probably won't come in until tomorrow.

**Comment by moxiegirl** [comment]
ping @SvenDowideit this one is kinda backing up can you take a look   @jfrazelle FYI

**Comment by SvenDowideit** [comment]
sadly, the DCO check isn't useful for the cherry-pick process.

however, you can deduce that if its cherry-picked from master, the change must already have DCO signature, so its all good 

so LGTM

File: docs/sources/project/make-a-contribution.md [hunk 1]
```diff
@@ -2,28 +2,78 @@ page_title: Make a project contribution
...
+    color: #fff;
+    border-radius: 2px;
+    box-shadow: inset 0 -1px 0 rgba(0,0,0,0.12);
+}
+
+.gh-label.black-belt  { background-color: #000000; color: #ffffff; }
+.gh-label.bug         { background-color: #fc2929; color: #ffffff; }
+.gh-label.improvement { background-color: #bfe5bf; color: #2a332a; }
+.gh-label.project-doc { background-color: #207de5; color: #ffffff; }
+.gh-label.white-belt  { background-color: #ffffff; color: #333333; }
```
Comment (SvenDowideit) [inline docs/sources/project/make-a-contribution.md]: mmm, I think this should be removed?

Comment (thaJeztah) [inline docs/sources/project/make-a-contribution.md]: these are added to mimic the github labels on the page :D and will only be used for that page,
so probably not interesting to put them in the central docs/base

**Comment by moxiegirl** [comment]
LGTM --- test local build looked fine.  As per email with @jfrazelle  I'm gonna push this.

**Pull request merged**